### PR TITLE
Bugs and new functionality for mint/burn multi-issuance tokens.

### PR DESCRIPTION
**Pull request by mpsandiford**

### CODE REVIEW

File: radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokenclasses/CreateTokenToParticlesMapper.java [hunk 1]
```diff
@@ -56,6 +56,12 @@
 				),
 				null
 		);
+
+		if (tokenCreation.getInitialSupply().isZero()) {
```
Comment (flotothemoon) [inline radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokenclasses/CreateTokenToParticlesMapper.java]: Why is this required now and was not necessary before?

File: radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/UnknownTokenException.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package com.radixdlt.client.application.translate.tokens;
+
+public class UnknownTokenException extends RuntimeException {
```
Comment (flotothemoon) [inline radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/UnknownTokenException.java]: Do we want to document stuff like exceptions now or at some later time?

File: radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokenclasses/CreateTokenToParticlesMapper.java (hunk 1 above)
Comment (mpsandiford) [reply radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokenclasses/CreateTokenToParticlesMapper.java]: Minting with initial supply never worked previously.  It should have for multi-issuance tokens.

File: radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/UnknownTokenException.java (hunk 2 above)
Comment (mpsandiford) [reply radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/UnknownTokenException.java]: I'll add some documentation.
